spring-cloud-stream/src/test/java/org/springframework/cloud/stream/function/FunctionSupportTests.java

```diff
@@ -99,13 +104,12 @@ public void testComposedFunctionIsAppliedToMessageSource() {
 		private FunctionProperties functionProperties;
 
 		@Bean
-		public IntegrationFlow messageSourceFlow(FunctionSupport<?, ?> functionSupport) {
+		public IntegrationFlow messageSourceFlow(FunctionSupport functionSupport) {
 			Supplier<Message<String>> messageSource = () -> MessageBuilder.withPayload("hello function")
 					.setHeader(MessageHeaders.CONTENT_TYPE, MimeTypeUtils.TEXT_PLAIN).build();
 
 			PollerMetadata pm = new PollerMetadata();
 			pm.setTrigger(new PeriodicTrigger(1, TimeUnit.SECONDS));
-			pm.setMaxMessagesPerPoll(1);
```

Comment:
Sorry, you didn't get me. The variable `pm` is fully out of use. I mean you don't use it in the next ` IntegrationFlows.from()` and actually you shouldn't. There is a good default one which fully enough for our use-case in this test.

Reply:
Daugh. . ..  ;(